### PR TITLE
Sort list and methods alphabetically

### LINKED ISSUE #4: Sort `fakerbot list` alphabetically
## Description

As it stands, calling 'fakerbot list' will list faker constants in an arbitrary order. This should be alphabetical.

### PR DESCRIPTION
**Pull request by amrrbakry**
Following our discussion about this #4

I'm not sure what is the best way to add a spec for this change. What do you think? @akabiru 

Closes #4

### CODE REVIEW

**Comment by amrrbakry** [comment]
@akabiru r?

**Comment by akabiru** [comment]
Hey @amrrbakry , sorry this has taken a while- I've just got wind of it now. 🙈 I'll have a look ASAP. 👍

File: lib/fakerbot/renderer.rb [hunk 1]
```diff
@@ -36,9 +36,10 @@ def call
     attr_reader :crayon, :hash, :options, :output, :pager, :screen, :tree
 
     def build_tree
-      hash.reduce({}) do |h, (faker, methods)|
-        h.merge! node(faker, methods)
+      data_tree = hash.reduce({}) do |h, (faker, methods)|
+        h.merge! node(faker, methods&.sort)
```
Comment (akabiru) [inline lib/fakerbot/renderer.rb:40]: Nice touch on safe navigation. 👍

**Review by akabiru — APPROVED** [review]
This looks great to me. Thanks for the good work @amrrbakry ! ✨🎉

File: lib/fakerbot/renderer.rb (hunk 1 above)
Comment (amrrbakry) [reply lib/fakerbot/renderer.rb:40]: thank you! :+1: specs wouldn't pass without it

Comment (akabiru) [reply lib/fakerbot/renderer.rb:40]: Yup, covers for the case when `methods` might be nil. I've also dropped in #11 . Good chance to increase the test coverage.

Comment (amrrbakry) [reply lib/fakerbot/renderer.rb:40]: good work :+1: :sparkles: :smile:

**Pull request merged**